coding/stylesheets.md

```diff
@@ -1,8 +1,12 @@
 ---
-title: BEM coding standard
+title: Stylesheets coding standard
 description: Code style standards for CSS and Sass
 ---
 
+## BEM methodology
```

Comment:
I think putting the BEM heading here is also too strong - again because these are our standard, not BEM's. Maybe leave this heading out so it just goes straight into the "introduction"? I would keep the line about "based on BEM..." but that seems quite sufficient.